parl/tests/gym.py

```diff
@@ -142,7 +142,8 @@ def get_action_meanings(self):
                 return ['NOOP'] * 6
 
         self.observation_space = ObservationSpace((210, 160, 3), 'unit8')
-        self.action_space = ActionSpace(6)
+        self.action_space = ActionSpace(n=6, shape=())
```

Comment:
Is the shape of action sapce of HalfCheetah-v1 just ()?

Reply:
add real obs/act space to mock env